app-backend/batch/src/main/scala/notification/NotifyIngestStatus.scala

```diff
@@ -103,170 +106,128 @@ final case class NotifyIngestStatus(sceneId: UUID)(
           """.trim.stripMargin
+          )
         )
     }
   }
 
   def createIngestEmailContentForOwner(
       pO: PlatformWithSceneOwner,
       scene: Scene,
-      ingestStatus: String): (String, String, String) = {
+      ingestStatus: String
+  ): (Subject, HtmlBody, PlainBody) = {
     val platformHost =
       pO.pubSettings.platformHost.getOrElse("app.rasterfoundry.com")
     ingestStatus match {
       case status: String if status == "INGESTED" =>
         (
-          s"${pO.platName}: Your scene ${scene.name} is ready to view",
-          s"""
+          Subject(s"${pO.platName}: Your scene ${scene.name} is ready to view"),
+          HtmlBody(s"""
           <html>
             <p>${pO.uName},</p><br>
             <p>The scene "${scene.name}" has been successfully ingested!</p>
             <p>If you have questions, support is available via in-app chat at ${platformHost} or less quickly via email to ${pO.pubSettings.emailSupport}.</p>
             <p>- The ${pO.platName} Team</p>
           </html>
-          """,
-          s"""
+          """),
+          PlainBody(
+            s"""
              | ${pO.uName}: The scene "${scene.name}" has been successfully ingested!
              | If you have questions, support is available via in-app chat at ${platformHost} or less quickly via email to ${pO.pubSettings.emailSupport}.
              | - The ${pO.platName} Team
           """.trim.stripMargin
+          )
         )
       case status: String if status == "FAILED" =>
         (
-          s"${pO.platName}: Your scene ${scene.name} failed to ingest",
-          s"""
+          Subject(s"${pO.platName}: Your scene ${scene.name} failed to ingest"),
+          HtmlBody(s"""
           <html>
             <p>${pO.uName},</p><br>
             <p>The scene "${scene.name}" has failed to ingest. </p>
             <p>If you have questions, support is available via in-app chat at ${platformHost} or less quickly via email to ${pO.pubSettings.emailSupport}.</p>
             <p>- The ${pO.platName} Team</p>
           </html>
-          """,
-          s"""
+          """),
+          PlainBody(s"""
              | ${pO.uName}: The scene "${scene.name}" has failed to ingest.
              | If you have questions,
              | support is available via in-app chat at ${platformHost} or less quickly via email to ${pO.pubSettings.emailSupport}.
              | - The ${pO.platName} Team
-          """.trim.stripMargin
+          """.trim.stripMargin)
         )
     }
   }
 
   def sendIngestStatusEmailToConsumers(
       platformsWithConsumers: List[PlatformWithUsersSceneProjects],
       scene: Scene,
-      ingestStatus: String) =
-    platformsWithConsumers.map(pU => {
-      val email = new NotificationEmail
-
+      ingestStatus: String
+  ): IO[Unit] =
+    (platformsWithConsumers traverse { pU =>
       (pU.getUserEmail, pU.pubSettings.emailIngestNotification) match {
-        case ("", true) =>
```

Comment:
same as above -- the email settings validation will not send emails in these cases